### ISSUE TITLE
Can not `$view()` on mac

### ISSUE THREAD

**Issue by schloerke** [rstudio/chromote#79]
@DivadNojnarg commented on https://github.com/rstudio/chromote/commit/b605b46b1e114468d09404734053e8ee8c3594a9#commitcomment-68872485

> @schloerke : this last update fails on my mac locally sh: /Applications/Google: No such file or directory.

**Comment by schloerke** [comment]
Related #73 - feature
Related #78 - breakage

**Comment by schloerke** [comment]
@DivadNojnarg , what is the print value for `chromote::default_chromote_object()$get_browser()`

My value on macOS w/ R 4.0:

```r
<Chrome>
  Inherits from: <Browser>
  Public:
    clone: function (deep = FALSE)
    close: function ()
    get_host: function ()
    get_path: function ()
    get_port: function ()
    get_process: function ()
    initialize: function (path = find_chrome(), args = get_chrome_args())
    is_local: function ()
  Private:
    finalize: function (e)
    host: 127.0.0.1
    path: /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
    port: 52671
    process: process, R6
```

**Comment by schloerke** [comment]
... turns out I didn't have the latest 🤦‍♂️ . 

With `main`, I could reprex and the bug fixed in #80
